### PR TITLE
Configure Firebase ci/cd

### PR DESCRIPTION
**Pull request by cafloyd**
Configuring Firebase CI/CD.

### CODE REVIEW

**Comment by cafloyd** [comment]
Hey @DwightTheShark ! This PR should set up CI/CD for your team's repo. You can see the github actions bot above added the preview link. Once this PR has been merged, your team's deployed app should point to `main` on this repo!

Feel free to merge this PR once you've approved to speed up the process!

**Review by DwightTheShark — APPROVED** [review]

**Pull request merged**
